src/google/cloud/ndb/model.py

```diff
@@ -594,6 +594,7 @@ def __init__(
         validator=None,
         verbose_name=None,
         write_empty_list=None,
+        **kwargs
```

Comment:
This is a bit ugly, but makes things work with Python 3.5 _and_ black, without sacrificing the new way to handle positional arguments. The alternative to this would be to revive the old positional decorator from utils and change the code to use it again. I prefer this fix, but still believe that dropping 3.5 would be the best solution (which also gives us f strings back).